docs/pages/docs/03-building-schema/defining-schema.md

```diff
@@ -198,6 +198,7 @@ Equivalent to `sub`, but only used in `relates` for defining the hierarchy of ro
 <div class="tab-content">
 <div role="tabpanel" class="tab-pane active" id="shell9">
 <pre class="language-graql"> <code>
+<!--test-ignore-->
```

Comment:
WHY?

Reply:
disabling temporarily, will ask around how to fix it tomorrow

Reply:
@kasper-piskorski FYI, the bug has been fixed by @duckofyork and the test has been uncommented